htmlvalidator/core.py

```diff
@@ -61,8 +61,7 @@ def _validate(html_file, encoding, args_kwargs):
             # 1 if it worked but there was validation errors/warnings
             raise ValidatorOperationalError(err)
 
-        # out = unicode(out, 'utf-8')
-        err = unicode(err, 'utf-8')
+        err = err.decode('utf-8')
```

Comment:
`.decode()` only works on bytestrings in Python 3. Is `err` a bytestring in Python 3?